build/scripts/tests/run_testworld.sh

```diff
@@ -2,7 +2,7 @@
 
 echo "Running Testworld"
 
-cleanup="ls testworld/hostconfigs/* | grep testworld | grep -v README.md | tr -d : | xargs rm -rf"
+cleanup="pkill -f testworld; ls testworld/hostconfigs/* | grep testworld | grep -v README.md | tr -d : | xargs rm -rf"
```

Comment:
What is the purpose of `pkill` here? Just kill a previous process if it was running? Or something else?

Reply:
it will kill everything testworld related on the previous run.

Reply:
haha including it self it seems :D